extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -404,17 +409,29 @@ function _jumpToMeasurement(
     displaySetInstanceUID
   );
 
-  const imageIdIndex = referencedDisplaySet.images.findIndex(
-    i => i.SOPInstanceUID === SOPInstanceUID
-  );
-
   // Todo: setCornerstoneMeasurementActive should be handled by the toolGroupManager
   //  to set it properly
   // setCornerstoneMeasurementActive(measurement);
 
   viewportGridService.setActiveViewportIndex(viewportIndex);
 
-  if (getEnabledElement(targetElement)) {
+  const enableElement = getEnabledElement(targetElement);
+  if (enableElement) {
+    // See how the jumpToSlice() of Cornerstone3D deals with imageIdx param.
+    const viewport = enableElement.viewport as IStackViewport | IVolumeViewport;
+    const { imageIds } = viewport;
+
+    const imageIdIndex = imageIds.findIndex(imageId => {
+      const {
+        SOPInstanceUID: SOPInstanceUID1,
+        frameNumber: frameNumber1,
+      } = getSOPInstanceAttributes(imageId);
+      return (
+        SOPInstanceUID1 === SOPInstanceUID &&
```

Comment:
I'm not a fan of appending 1 to the end of these variables, I prefer `aSOPInstanceUID` and `aFrameNumber`

Also use `===` instead of `==`

Reply:
done in 471e9c6a294d746a6b81bc6c92e19233c677c6aa